lib/cryptoexchange/exchanges/bitshares_assets/services/market.rb

```diff
@@ -0,0 +1,35 @@
+module Cryptoexchange::Exchanges
+  module BitsharesAssets
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
```

Comment:
Because https://cryptofresh.com/api/asset/markets?asset=BTS returns a list of tickers, this would be a `false` example right?